src/Files.Uwp/Helpers/WindowDecorationsHelper.cs

```diff
@@ -20,6 +20,7 @@ public static void RequestWindowDecorationsAccess()
                 {
                     "Files" => "xnYLj99c3vN9jFCZiDC6Rg==",
                     "FilesDev" => "Yoz9y1X66micskUKhFrJ5A==",
+                    "FilesPreview_wvne1zexy08sa" => "jz4pZSdYrC05IyNfTaNwZQ==",
```

Comment:
This is the package family name not the package name, the correct value should be `FilesPreview`

Reply:
Good catch, this should be fixed in the latest commit.